### PR TITLE
STORM-2277: Support given for storm-druid connector shaded jar 

### PR DESCRIPTION
**Pull request by omkreddy**
Support given for storm-druid connector shaded jar and added some debug logs to DruidBolt

### CODE REVIEW

File: external/storm-druid/pom.xml [hunk 1]
```diff
@@ -40,16 +40,26 @@
...
+                  <groupId>mysql</groupId>
+                  <artifactId>mysql-connector-java</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
             <version>2.11.8</version>
+            <scope>provided</scope>
```
Comment (HeartSaVioR) [inline external/storm-druid/pom.xml]: Should users include scala-language manually? Then it would be better to be documented.

**Comment by HeartSaVioR** [comment]
Looks good to me overall. Left minor comment.

File: external/storm-druid/pom.xml (hunk 1 above)
Comment (omkreddy) [inline external/storm-druid/pom.xml]: @HeartSaVioR  Thanks for the review.  Yes, Scala dependency can be provided by users. Updated the document.

**Comment by HeartSaVioR** [comment]
+1

**Pull request merged**